docs/paper/paper.md

```diff
@@ -0,0 +1,99 @@
+title: 'astroplan: an astropy affiliated package for observation planning'
+tags:
+  - astronomy
+  - Python
+  - astropy-affiliated
+authors:
+ - name: Brett M. Morris
+   orcid: 0000-0003-2528-3409
+   affiliation: 1
+ - name: Karl Vyhmeister
+   orcid: 
+   affiliation: 2
+ - name: Stuart Littlefair
+   orcid: 0000-0001-7221-855X
+   affiliation: 3
+ - name: Brigitta Sipocz
+   orcid: 
+   affiliation: 4
+ - name: Erik Tollerud
+   orcid: 0000-0002-9599-310X
+   affiliation: 5
+ - name: Jazmin Berlanga Medina
+   orcid: 
+   affiliation: 6
+ - name: Adrian Price-Whelan
+   orcid: 0000-0003-0872-7098
+   affiliation: 7
+ - name: Wilfred Tyler Gee
+   orcid: 0000-0002-2931-7605
+   affiliation: 8
+ - name: Stephanie Douglas
+   orcid: 0000-0001-7371-2832
+   affiliation: 9
+ - name: Aniket Pandey
+   orcid: 
+   affiliation: 10
+ - name: Larry Bradley
+   orcid: 0000-0002-7908-9284
+   affiliation: 5
+ - name: Thomas Robitaille
+   orcid: 0000-0002-8642-1329
+   affiliation: 11
+ - name: Eric Jeschke 
+   orcid: 0000-0001-9586-4482
+   affiliation: 12
+affiliations:
+ - name: University of Washington
+   index: 1
+ - name: California Institute of Technology
+   index: 2
+ - name: University of Sheffield
+   index: 3
+ - name: University of Hertfordshire
+   index: 4
+ - name: Space Telescope Science Institute
+   index: 5
+ - name: Purdue University
+   index: 6
+ - name: Princeton University
+   index: 7
+ - name: Macquarie University
+   index: 8
+ - name: Harvard-Smithsonian Center for Astrophysics
+   index: 9
+ - name: Indian Institute of Technology Kanpur
+   index: 10
+ - name: (freelance)
+   index: 11
+ - name: National Astronomical Observatory of Japan 
+   index: 12
+
+date: 2 September 2017
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Astropy is the core Python package for fundamental astronomical computations. An 
+ecosystem of astropy affiliated packages provides efficient access to more 
+specific astronomical tasks. We present astroplan – an astropy affiliated 
```

Comment:
I wonder if you want to swap the order of this paragraph?  E.g. structure it more like "Astroplan does x.  It is an astropy affiliated package, which is Y".  That makes it clearer that this is focused on *astroplan* rather than the wider ecosystem.  (The actual content is great though - just suggesting it be flipped around)